src/main/resources/openapi/audio.yaml

```diff
@@ -29,6 +29,31 @@ paths:
             schema:
               $ref: '#/components/schemas/AudioRequestDetails'
         description: Audio Request item to add
+    delete:
+      tags:
+        - Audio
+      summary: Delete Audio Request and Processed Audio
+      description: Delete Audio Request and Processed Audio
+      operationId: deleteAudioRequest
+      parameters:
+        - in: query
+          name: audioRequestId
```

Comment:
Query params should be lower snake case
but I think this should really be a path param to be similar to `DELETE
[/courthouses/{courthouse_id}](http://localhost:4550/swagger-ui/index.html#/Courthouses/courthousesCourthouseIdDelete)`